src/remote/impl/rnp.external.controller.service.ts

```diff
@@ -0,0 +1,35 @@
+import { HttpService } from '@nestjs/common';
+import { Logger } from 'protocol-common/logger';
+import { ProtocolHttpService } from 'protocol-common/protocol.http.service';
+import { IExternalControllerService } from '../external.controller.service.interface';
+
+/*
+    Controller Service for honduras implementation
```

Comment:
There should be nothing specific to the Honduras implementation in this repository. This is an open source, generic, non-implementation-specific repository. If we need anything that is specific to an implementation, it needs to go somewhere else and we need to figure out the proper abstraction.